backend/Dockerfile

```diff
@@ -1,14 +1,32 @@
-FROM golang:1.10.2-stretch
+FROM golang:1.10.2
```

Comment:
please explain this change

Reply:
"golang:1.10.2" is a multi-arch image, which means it will build for amd64 & ppc64le as well. Hence the change. Sorry for not adding the description at the time of commit. I will make sure I will do that now on wards.